### ISSUE TITLE
'script' and 'game' globals do not account for indexing and calling methods on fields

### ISSUE THREAD

**Issue by Nimblz** [Quenty/luacheck-roblox#1]
```lua
require(script.Parent.SomeModule)
require(script.SomeChildModule)
require(script.Parent:WaitForChild("SomeModule"))
local SomeInstance = game.Workspace.SomeInstance
local AnotherInstance = script.ChildInstance
```

These potentially valid pieces of code will raise warnings that they access undefined fields.

**Comment by Nimblz** [comment]
I've created a pull request where this issue is fixed
https://github.com/Quenty/luacheck-roblox/pull/2

**Comment by Nimblz** [comment]
I've created a pull request that fixes this issue #2

**Comment by Quenty** [comment]
Closed by #2
